ethereum/sidecar/cli/flags.go

```diff
@@ -61,5 +66,19 @@ func NewFlagSetEthereumSidecar(
 		"Requests per minute for an Ethereum RPC provider",
 	)
 
+	fs.String(
+		FlagBridgeOutServerAddress,
+		defaultBridgeOutServerAddress,
+		"Address of the gRPC bridge-out server. The bridge-out server is "+
+			"run by validators as part of mezod and responds with "+
+			"AssetsUnlock events",
```

Comment:
Is the concatenation here to make a multiline string? I'd use ` strings instead then for readability.

Reply:
I wrote it that way to avoid creating a vary long line of code (say `80` chars in length).